sqlitecluster/SQLiteCommand.h

```diff
@@ -69,5 +69,5 @@ class SQLiteCommand {
     SQLiteCommand& operator=(SQLiteCommand&& from) = default;
 
     // Destructor.
-    ~SQLiteCommand() {}
+    virtual ~SQLiteCommand() {}
```

Comment:
It's a flat out bug that this wasn't `virtual` before that we got around solely by luck.